pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -52,8 +52,8 @@ type NamespacedResourcesFunc func() ([]*metav1.APIResourceList, error)
 // that may require quota to be recalculated.
 type ReplenishmentFunc func(groupResource schema.GroupResource, namespace string)
 
-// ResourceQuotaControllerOptions holds options for creating a quota controller
-type ResourceQuotaControllerOptions struct {
+// QuotaControllerOptions holds options for creating a quota controller
+type QuotaControllerOptions struct {
```

Comment:
```suggestion
type ControllerOptions struct {
```

https://github.com/golang/go/wiki/CodeReviewComments#package-names